### DISCUSSION TITLE
Most federated server posts don't appear in timeline (only followed accounts)

### DISCUSSION THREAD

**Discussion by JetForMe** [mastodon/mastodon#21751] [Q&A] [answered]
I've got a small instance up and running the last few days. Most things seem to work fine, but a large instance (mastodon.social), while evidently federated, doesn't appear in the federated timeline on my local server. Posts from followed accounts do, but not the firehose.

I don't block or limit any domains, and I don't see anything obvious in the logs. Any advice on how to troubleshoot this?

**Comment by lifeofguenter** [comment, marked as answer]
That would require mastodon.social to make use of an [relay](https://github.com/brodi1/activitypub-relays) , which they are not currently transparent about.

**Comment by JetForMe** [comment]
Ah, thank you. I just learned about relays a short while ago. I misunderstood this about Mastodon; I assumed that federation meant seeing everything from servers you linked to.
